dockerfiles/drbd-driver-loader/Dockerfile.bullseye

```diff
@@ -2,10 +2,10 @@ FROM debian:bullseye
 
 ARG DRBD_VERSION
 
-RUN apt-get update && apt-get upgrade -y && apt-get install -y kmod gnupg wget make gcc patch elfutils curl && apt-get clean
+RUN apt-get update && apt-get upgrade -y && apt-get install -y kmod gnupg wget make gcc patch elfutils curl linux-kbuild-* && apt-get clean
```

Comment:
this should be quoted: `apt install ... 'linux-kbuild-*'` otherwise - very theoretically - it gets shell expanded.